ports_module/cortex_a7/gnu/module_manager/src/tx_thread_stack_build.s

```diff
@@ -38,6 +38,8 @@ CPSR_MASK       =       0xFF                    // Mask initial CPSR, T, IRQ & F
 CPSR_MASK       =       0xBF                    // Mask initial CPSR, T, IRQ interrupts enabled
 #endif
 
+    .global     _tx_thread_schedule
```

Comment:
Should `.global` be replaced with `EXTERN` here?

Reply:
[ports_module/cortex_a7/gnu/module_manager/src/tx_thread_context_save.s] has been fixed